hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/json/JacksonDeserializer.java

```diff
@@ -377,6 +467,30 @@ public static Condition deserializeCondition(JsonNode node) throws JsonProcessin
         return map;
     }
 
+    public static class ConditionException extends JsonProcessingException {
```

Comment:
Should this be @ApplicationException to avoid being wrapped in EJBException if thrown?